### PR TITLE
Feature added: Forgot Password

### PR DESCRIPTION
**Pull request by codingwill**
I have created forgot password by adding several classes and functions as follows:

File(s) added/modified:
- app/Http/Controllers/**ForgotPasswordController.php**
- app/Mail/**ForgotEmail.php**
- app/Models/**VerificationTokens.php**
- resources/views/**resetPassword.blade.php**
- routes/**api.php**

Features added/modified:
- added forgot password endpoint that will generate reset password token and send it by customer's email.

### CODE REVIEW

File: app/Http/Controllers/ForgotPasswordController.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    // To send forgot password link to customers email (with verification token)
+    public function forgotPassword(Request $request) {
+        // Retrive user info by their email
+        $userModel = Customers::where('email', $request->email)
+                            ->first();
+
+        // If email does not exist
+        if (!$this->validateEmail($request->email)) {
```
Comment (cakazies) [inline app/Http/Controllers/ForgotPasswordController.php]: Vedanta ini sama yang atas apa ??

Comment (codingwill) [reply app/Http/Controllers/ForgotPasswordController.php]: Oh iya sorry mas, sebetulnya sama aja, jadinya malah manggil query yang sama 2x yah.
Tapi udah aku fix di branch fixing, segera di PR secepatnya ya.

Comment (codingwill) [reply app/Http/Controllers/ForgotPasswordController.php]: Atau lebih baik commit di branch ini aja ya mas untuk revisinya?

Comment (codingwill) [reply app/Http/Controllers/ForgotPasswordController.php]: Ini barusan aku coba pushed ke branch ini mas, kira-kira gimana ya mas?

Comment (cakazies) [reply app/Http/Controllers/ForgotPasswordController.php]: oke will check

Comment (cakazies) [reply app/Http/Controllers/ForgotPasswordController.php]: kalau ada comment kya gini , push di branch yang Sama yah

**Review by cakazies — APPROVED** [review]

**Pull request merged**